webpack.config.js

```diff
@@ -30,20 +32,15 @@ module.exports = {
           plugins: ['transform-runtime']
         }
       },
-      { test: /\.css$/,
-        include: path.resolve(__dirname, 'static_src/css'),
+      {
+        test: /\.css$/,
+        include: [
+          path.resolve(__dirname, 'static_src/css'),
+          path.resolve(__dirname, 'node_modules/cloudgov-style'),
+          CG_STYLE_PATH
```

Comment:
oh I mean also in the use case where we have a pinned cloudgov-style rather the the symlinked folder